lms/djangoapps/courseware/tests/test_view_authentication.py

```diff
@@ -273,6 +287,8 @@ def test_global_staff_access(self):
         """
         self.login(self.global_staff_user)
 
+        self.grant_sudo_access(self.course.id.to_deprecated_string(), 'test')
+        self.grant_sudo_access(self.test_course.id.to_deprecated_string(), 'test')
         # and now should be able to load both
         urls = [reverse('instructor_dashboard', kwargs={'course_id': self.course.id.to_deprecated_string()}),
                 reverse('instructor_dashboard', kwargs={'course_id': self.test_course.id.to_deprecated_string()})]
```

Comment:
It would be good to add a test that without sudo access, you can't access the instructor dashboard.